app/models/master_file.rb

```diff
@@ -137,6 +139,16 @@ class MasterFile < ActiveFedora::Base
   end
   # validates :file_format, presence: true, exclusion: { in: ['Unknown'], message: "The file was not recognized as audio or video." }
 
+  validate :validate_for_staleness
+
+  def validate_for_staleness
+    # Created a long time ago but did not get encoded?
+    if create_date.present? && create_date < STALENESS_CUTOFF.ago &&
```

Comment:
I'm guessing the `.ago` is down here because we want to calculate it when this method runs and not at constant definition time.

Honestly, it might be more readable to just have `1.week.ago` inline? Or since a similar constant is used elsewhere that might want to be shared, pull the entire validation up into its own custom validation shared by both models, like we did here https://github.com/ualbertalib/jupiter/blob/master/app/validators/uri_validator.rb

Reply:
I'll just hard code the `1.week.ago`. I think trying to abstract this so that both situations can use it (while handling the differences) will leave something that is ultimately harder to read.